README.md

```diff
@@ -1,11 +1,121 @@
-# Stackable Operator for Apache Kafka
+<p align="center">
+  <img width="150" src="./.readme/static/borrowed/Icon_Stackable.svg" alt="Stackable Logo"/>
+</p>
+
+<h1 align="center">Stackable Operator for Apache Kafka</h1>
 
-[![Build Actions Status](https://ci.stackable.tech/job/kafka%2doperator%2dit%2dnightly/badge/icon?subject=Integration%20Tests)](https://ci.stackable.tech/job/kafka%2doperator%2dit%2dnightly)
 [![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://GitHub.com/stackabletech/kafka-operator/graphs/commit-activity)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-green.svg)](https://docs.stackable.tech/home/stable/contributor/index.html)
+[![License OSL3.0](https://img.shields.io/badge/license-OSL3.0-green)](./LICENSE)
+
+[Documentation](https://docs.stackable.tech/kafka-operator/stable/index.html) | [Stackable Data Platform](https://stackable.tech/) | [Platform Docs](https://docs.stackable.tech/)
+
+This is a Kubernetes operator to manage [Apache Kafka](https://kafka.apache.org/) clusters.
+
+It is part of the Stackable Data Platform, a curated selection of the best open source data apps like Kafka, Druid, Trino or Spark, [all](#other-operators) working together seamlessly. Based on Kubernetes, it runs everywhere – [on prem or in the cloud]((#supported-platforms)).
+
+## Installation
+
+You can install the operator using [stackablectl or helm](https://docs.stackable.tech/kafka/stable/getting_started/installation.html).
+
+Read on to get started with it, or see it in action in one of our [demos](https://stackable.tech/en/demos/).
+
+## Getting Started
+
+You can follow this [tutorial](https://docs.stackable.tech/kafka/stable/getting_started/first_steps.html) to create a Kafka cluster with three replicas.
+
+## Documentation
+
+The documentation for this operator can be found [here](https://docs.stackable.tech/kafka/stable/index.html).
+
+The documentation for all Stackable products can be found at [docs.stackable.tech](https://docs.stackable.tech).
+
+## What Does This Do?
+
+Operator looks at custom resources in the cluster, and makes sure apps are started according to those custom resources.
+
+If those resources change, the operator makes sure that the right actions are taken so the app is the way you want it to be.
+
+Here's what a custom resource for a Kafka cluster can look like:
+
+```
+---
+apiVersion: kafka.stackable.tech/v1alpha1
+kind: KafkaCluster
+metadata:
+  name: simple-kafka
+spec:
+  version: 3.2.0-stackable0.1.0
+  zookeeperConfigMapName: simple-kafka-znode
+  config:
+    tls: null
+  brokers:
+    roleGroups:
+      default:
+        replicas: 3
+```
+
+## Behind the Scenes
+
+This operator is written by [Stackable](https://www.stackable.tech) in Rust.
+
+It uses kube-rs to talk to Kubernetes.
```

Comment:
IMO this sounds extremely colloquial to me. It's something we say not something we write ;) A suggestion:
```suggestion
This operator uses [kube.rs](https://kube.rs/) to interact with the Kubernetes control plane.
```